server/server.ts

```diff
@@ -1,6 +1,11 @@
 import express from 'express';
 import { exampleRouter } from './routes/example';
+import { eventRouter } from './routes/event';
+import { financialRouter } from './routes/financial';
+import { receiptRouter } from './routes/receipt';
+
 import cors from 'cors';
+import { isFunctionExpression } from 'typescript';
```

Comment:
Is this necessary?

Reply:
Fixed. That was added in unexpectedly by intellisense.